src/main/java/org/icgc/argo/workflow_management/wes/NextflowService.java

```diff
@@ -148,17 +148,15 @@ private String startRun(RunParams params) {
    *
    * @return the kube client to be used to interact with deployed workflow pods
    */
-  private DefaultKubernetesClient createWorkflowRunK8sClient() {
+  private DefaultKubernetesClient createWorkflowRunK8sClient(String context, String masterUrl) {
     try {
-      val masterUrl = config.getK8s().getMasterUrl();
       val runNamespace = config.getK8s().getRunNamespace();
       val trustCertificate = config.getK8s().isTrustCertificate();
-      val config =
-          new ConfigBuilder()
-              .withTrustCerts(trustCertificate)
-              .withMasterUrl(masterUrl)
-              .withNamespace(runNamespace)
-              .build();
+      Config config= Config.autoConfigure(context);
```

Comment:
The missing space between `config=` means that the code formatter wasn't run. The code base has the [spotify code formatter plugin ](https://github.com/spotify/fmt-maven-plugin) (formerly from coveo, [our pom](https://github.com/icgc-argo/workflow-management/blob/develop/pom.xml#L197) is old); this can be run with `mvn process-sources`, the code formatter will trigger during that phase.

Reply:
ran the plugin.